std_msgs/Header header

zcm2ros/RadarDetectionArray detections
zcm2ros/RadarTrackArray tracks
zcm2ros/RadarInfo[] info